### ISSUE TITLE
On Firefox, configuration data cannot imported from Brave

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#9247]
**Describe the issue you encountered:** On Firefox, configuration data cannot imported from Brave. Brave is not recognized as a browser from which data can be imported.

![clipboard01](https://cloud.githubusercontent.com/assets/3362943/26759216/074c0d9e-4933-11e7-9baa-39a6f3468b02.jpg)

- Brave Version (revision SHA): 0.15.314

- Steps to reproduce:
    1. Install Firefox
    2. Install Brave
    3. Import Settings and Data from bookmark manager on Firefox

- Expected result: `Brave` should be displayed there

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

**Comment by Liunkae** [comment]
Is this something that needs to be fixed by Firefox? I assume Firefox does not understand the format of Brave's config data.
